charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -153,5 +153,5 @@ receivers:
     enabled: true
 
 # -- (bool) Send only the [metrics required](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/docs/metrics-lowDataMode.md) to light up the NR kubernetes UI, this agent defaults to setting lowDataMode true, but if this setting is unset, lowDataMode will be set to false
```

Comment:
After this change, wouldn't `lowDataMode` be true if unset? If so, we should update this wording.

Reply:
@kang-makes Bumping this :) Thanks!